src/components/Carousel.js

```diff
@@ -589,7 +591,9 @@ class Carousel extends Component {
                 currentSlideIndex={this.getActiveSlideIndex()}
                 index={index}
                 width={this.getCarouselElementWidth()}
+                widthUnit={this.props.itemWidthUnit || 'px'}
                 offset={index !== slides.length ? this.props.offset : 0}
+                offsetUnit={this.props.offsetUnit || 'px'}
```

Comment:
please use defaultProps instead

Reply:
fixed in be70f5a98c2ea1a97573d027e8d0c03c394d527f